### PR TITLE
Fix maximum entries in polynomial interpolation approach

### PR DESCRIPTION
**Pull request by sifnoc**
To address the issue #274 

- Added advice column in the range check circuit
- Assign zero value from instance value
- Replace terms `polynomial encoding` with `polynomial interpolation`

### CODE REVIEW

File: kzg_prover/src/chips/range/tests.rs [hunk 1]
```diff
@@ -130,6 +131,12 @@ impl Circuit<Fp> for TestCircuit {
 
         let add_selector = meta.selector();
 
+        let instance = meta.instance_column();
+        meta.enable_equality(instance);
+
+        let z0 = meta.advice_column();
```
Comment (alxkzmn) [inline kzg_prover/src/chips/range/tests.rs:137]: Seems like z0 is not used anymore

Comment (sifnoc) [reply kzg_prover/src/chips/range/tests.rs]: Thanks, I quickly fixed! at [c831f5d](https://github.com/summa-dev/summa-solvency/pull/276/commits/c831f5d2f8e28adb0eed0de593eb3c93ddc0ecd7)

**Review by alxkzmn — CHANGES_REQUESTED** [review]
Looks good, you can merge after removing the unused z0 advice column. Thanks for cleaning up the naming!

**Review by alxkzmn — APPROVED** [review]
LGTM

**Pull request merged**